libs/platform/unit/platform_unit_test.cpp

```diff
@@ -1063,3 +1069,39 @@ TEST_CASE("get_authentication_id", "[platform]")
 
     REQUIRE(ebpf_platform_get_authentication_id(&authentication_id) == EBPF_SUCCESS);
 }
+
+// See https://en.wikipedia.org/wiki/Chi-squared_test for details.
+#define SEQUENCE_LENGTH 100000000
+#define NUM_BINS 65536
+#define SIGNIFICANCE_LEVEL 0.05
```

Comment:
nit: it is not being used anywhere (maybe was offline used to calculate the critical value). Should we change this to a comment?